cmd/minikube/cmd/start.go

```diff
@@ -135,6 +135,9 @@ func runStart(cmd *cobra.Command, args []string) {
 	}
 
 	displayEnviron(os.Environ())
+	if viper.GetBool(force) {
+		out.WarningT("minikube skips various validations when --force is supplied; this may lead to unexpected behavior")
```

Comment:
add dot at the end.

Reply:
I think that would look odd as none of the other logs have periods at the end (sample output is in the PR description)